mcp/src/minecraft/com/elementfx/tvp/ad/item/crafting/RecipesCustom.java

```diff
@@ -10,5 +11,6 @@
 	public void addRecipes(CraftingManager manager) 
 	{
 		manager.addRecipe(new ItemStack(Items.PIPE, 1), new Object[] {"  S", "SS ", "S  ", 'S', Items.STICK});
+		manager.addShapelessRecipe(new ItemStack(Items.THROWING_STONE, 4), new Object[] {Blocks.COBBLESTONE});
```

Comment:
Oops, it seems I accidentally grabbed the Cobblestone Monster Egg block instead 🤦‍♂️. Works now.